doc/configuration.rst

```diff
@@ -1289,6 +1298,46 @@ further deferred, if desired).  The following produces only one plot::
   plt.plot([2, 2])
   plt.show()
 
+.. _hiding_code_blocks:
+
+Hiding code blocks
```

Comment:
I wonder if 'code lines' would be better?

Reply:
Agreed, changed to `Hiding lines of code`